support-frontend/assets/pages/supporter-plus-landing/components/paymentTsAndCs.tsx

```diff
@@ -211,19 +231,94 @@ export function PaymentTsAndCs({
 		<div css={container}>
 			<FinePrint mobileTheme={mobileTheme}>
 				{inTier3 && (
-					<ThreeTierTerms
+					<TierThreeTerms
 						paymentFrequency={contributionType === 'ANNUAL' ? 'year' : 'month'}
 					/>
 				)}
 				{inSupporterPlus &&
-					copyAboveThreshold(
-						contributionType,
-						productNameAboveThreshold,
-						'SupporterPlus',
-						promotion,
-					)}
-				{inSupport && copyBelowThreshold(contributionType)}
+					copyAboveThreshold(contributionType, productKey, promotion)}
+				{inSupport && copyBelowThreshold(contributionType, countryGroupId)}
 			</FinePrint>
 		</div>
 	);
 }
+
+export function SummaryTsAndCs({
+	contributionType,
+	countryGroupId,
+	currency,
+	amount,
+	amountIsAboveThreshold,
+	productKey,
+	cssOverrides,
+}: PaymentTsAndCsProps): JSX.Element {
+	const inTier3 = productKey === 'TierThree' && amountIsAboveThreshold;
+	const inTier2 = productKey === 'SupporterPlus' && amountIsAboveThreshold;
+	const inTier1 = productKey === 'Contribution' || !(inTier2 || inTier3);
+
+	const amountCopy = ` of ${formatAmount(
+		currencies[currency],
+		spokenCurrencies[currency],
+		amount,
+		false,
+	)}`;
+
+	const copyTier1 = (contributionType: ContributionType) => {
+		return (
+			<>
+				<div>
+					We will attempt to take payment{amountCopy},{' '}
+					<TsAndCsRenewal contributionType={contributionType} />, from now until
+					you cancel your payment. Payments may take up to 6 days to be recorded
+					in your bank account. You can change how much you give or cancel your
+					payment at any time.
+				</div>
+			</>
+		);
+	};
+
+	const copyTier2 = (
+		contributionType: ContributionType,
+		productKey: ProductKey,
+	) => {
+		return (
+			<>
+				<div>
+					The {productCatalogDescription[productKey].label} subscription and any
+					contribution will auto-renew each{' '}
+					{frequencySingular(contributionType)}. You will be charged the
+					subscription and contribution amounts using your chosen payment method
+					at each renewal, at the rate then in effect, unless you cancel.
+				</div>
+			</>
+		);
+	};
+
+	const copyTier3 = (
+		contributionType: ContributionType,
+		productKey: ProductKey,
+	) => {
+		return (
+			<>
+				<div>
+					The {productCatalogDescription[productKey].label} subscriptions will
```

Comment:
Nice to see this becoming the source of truth!

Reply:
Thanks, great recommend to pass `productKey` down instead